addons/web/controllers/utils.py

```diff
@@ -216,3 +216,12 @@ def _local_web_translations(trans_file):
                                   or WEB_TRANSLATION_COMMENT in x.auto_comments):
             messages.append({'id': x.id, 'string': x.string})
     return messages
+
+class Utils(Controller):
+
+    @route('/web/utils/check_record_company', type='json', auth="user")
+    def check_record_company(self, model, res_id, company_ids):
+        company_dependant = hasattr(request.env[model], 'company_id')
+        if company_dependant:
+            return bool(request.env[model].sudo().search_count([('id', '=', res_id), ('company_id', 'in', company_ids)], limit=1))
+        return True
```

Comment:
You'll find other controller utilities inside the `webclient` python module at `addons/web/controllers/webclient.py` so I would suggest you define that module there under the name `/web/webclient/check_record_company`.

Reply:
aright !

Reply:
done !